content/03.methods.md

```diff
@@ -19,5 +19,18 @@
 ### Gene Expression Abundance Estimation
 
 ### RNA Fusion Calling and Prioritization
```

Comment:
The section is written as passive voice. Can you write it as active voice? We ran... instead of XYZ was run.

Reply:
fixed